sql/core/benchmarks/StateStoreBasicOperationsBenchmark-jdk21-results.txt

```diff
@@ -2,141 +2,110 @@
-
-OpenJDK 64-Bit Server VM 21.0.2+13-LTS on Linux 6.5.0-1016-azure
-AMD EPYC 7763 64-Core Processor
-merging 10000 rows with 10 values per key (10000 rows to overwrite - rate 100):  Best Time(ms)   Avg Time(ms)   Stdev(ms)    Rate(M/s)   Per Row(ns)   Relative
---------------------------------------------------------------------------------------------------------------------------------------------------------------
-RocksDB (trackTotalNumberOfRows: true)                                                    519            533           7          0.0       51916.6       1.0X
-RocksDB (trackTotalNumberOfRows: false)                                                   171            177           3          0.1       17083.9       3.0X
-
-OpenJDK 64-Bit Server VM 21.0.2+13-LTS on Linux 6.5.0-1016-azure
-AMD EPYC 7763 64-Core Processor
-merging 10000 rows with 10 values per key (5000 rows to overwrite - rate 50):  Best Time(ms)   Avg Time(ms)   Stdev(ms)    Rate(M/s)   Per Row(ns)   Relative
-------------------------------------------------------------------------------------------------------------------------------------------------------------
-RocksDB (trackTotalNumberOfRows: true)                                                  506            521           7          0.0       50644.0       1.0X
-RocksDB (trackTotalNumberOfRows: false)                                                 170            176           3          0.1       17022.0       3.0X
-
-OpenJDK 64-Bit Server VM 21.0.2+13-LTS on Linux 6.5.0-1016-azure
-AMD EPYC 7763 64-Core Processor
-merging 10000 rows with 10 values per key (1000 rows to overwrite - rate 10):  Best Time(ms)   Avg Time(ms)   Stdev(ms)    Rate(M/s)   Per Row(ns)   Relative
-------------------------------------------------------------------------------------------------------------------------------------------------------------
-RocksDB (trackTotalNumberOfRows: true)                                                  493            508           6          0.0       49319.3       1.0X
-RocksDB (trackTotalNumberOfRows: false)                                                 169            175           3          0.1       16897.6       2.9X
-
-OpenJDK 64-Bit Server VM 21.0.2+13-LTS on Linux 6.5.0-1016-azure
-AMD EPYC 7763 64-Core Processor
-merging 10000 rows with 10 values per key (0 rows to overwrite - rate 0):  Best Time(ms)   Avg Time(ms)   Stdev(ms)    Rate(M/s)   Per Row(ns)   Relative
---------------------------------------------------------------------------------------------------------------------------------------------------------
-RocksDB (trackTotalNumberOfRows: true)                                              495            508           6          0.0       49462.5       1.0X
-RocksDB (trackTotalNumberOfRows: false)                                             169            175           3          0.1       16896.6       2.9X
```

Comment:
Could you double-check this, @neilramaswamy ? This file looks manually edited to me.

Reply:
Thank you!